nautobot/core/views/mixins.py

```diff
@@ -414,7 +414,7 @@ def form_valid(self, form):
         else:
             # render the form with the error message.
             data = {}
-            if self.action in ["bulk_update", "bulk_destroy"]:
+            if not request.POST.get("_all") and self.action in ["bulk_update", "bulk_destroy"]:
```

Comment:
Aren't we going to need similar code changes for bulk-update views/templates? IIRC they have the same set of problems with rendering the full table of impacted objects.

Reply:
Implementing `BulkEdit` is more complex than `BulkDelete` since each view relies on a custom form rather than a generic one. Additionally, forms currently accept only a `pk_list` rather than a full `queryset`, which would require additional changes to accommodate. Given the scope, I’ll defer this implementation for a future PR.

Reply:
https://github.com/nautobot/nautobot/issues/6465